AutoscoperM/AutoscoperMLib/IO.py

```diff
@@ -262,8 +263,8 @@ def _createNewVolumeNode(nodeName: str) -> slicer.vtkMRMLVolumeNode:
     volumeNode.SetIJKToRASDirections(imageDirections)
     volumeNode.SetAndObserveImageData(imageData)
     volumeNode.CreateDefaultDisplayNodes()
-    volumeNode.CreateDefaultStorageNode()
-    return volumeNode
+    defaultStorageNode = volumeNode.CreateDefaultStorageNode()
```

Comment:
Since the function `CreateDefaultStorageNode()`[^1] create the node without adding it to the scene (behavior is different from `CreateDefaultDisplayNodes()`[^2]), there is no need to call the function.

I then suggest to simplify the function `_createNewVolumeNode()` by removing the call to `CreateDefaultStorageNode()`

[^1]: https://github.com/Slicer/Slicer/blob/f898e9efc7728dd06bbab5da65e5d15e4a2fc79a/Libs/MRML/Core/vtkMRMLScalarVolumeNode.cxx#L144-L155
[^2]: https://github.com/Slicer/Slicer/blob/f898e9efc7728dd06bbab5da65e5d15e4a2fc79a/Libs/MRML/Core/vtkMRMLScalarVolumeNode.cxx#L158-L179